docs/get-started/data-collection.md

```diff
@@ -195,7 +239,43 @@ result = (
 towhee.plot(query.to_list(), result.to_list())
 ```
 
+## Prepare LMDB files for Pytorch Training
+
+```python
+>>> DataCollection.from_glob('path_train/*.jpg') \
+			.set_parallel(num_worker=5) \
+			.load_image() \
+  		.random_resize_crop(224) \
+      .random_horizontal_flip() \
+      .to_tensor() \
+      .save('./train.lmdb')
+```
+
+## Named Entity Extraction
+
+```python
+>>> dc = DataCollection.read_text('path_data/data.json') \
```

Comment:
`from_glob()` for `load_image()`, but `read_text()` for `load_json()`,  is there a need for unity here?

Reply:
I think we should add mixins to API documents too.